### PR TITLE
Bazaar redirect for searched item

### LINKED ISSUE #183: Bazaar
**Bazaar information**
* [x] Display bazaar's value. Added by #191 .
* [x] Highlight the item that redirected from Item Market. Added by #196 .

### PR DESCRIPTION
**Pull request by Sashank999**
Closes #183 .

### CODE REVIEW

**Comment by Sashank999** [comment]
IBF agreed.

**Comment by Sashank999** [comment]
> Would it be possible to fill in the search bar (might have to be cleared with IBF first)?

Even thought IBF agreed, their React JS wont.

**Comment by Sashank999** [comment]
> Would it be possible to fill in the search bar (might have to be cleared with IBF first)?

Even though IBF agreed, their React JS wont.

File: extension/scripts/features/itemredirect/ttBazaarRedirect.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+				) {
+					foundItem = true;
+					item.classList.add("tt-flash");
+					item.scrollIntoView({ behavior: "smooth" });
+					break;
+				}
+			}
+
+			if (!foundItem) {
+				document.find(".info-msg-cont .msg").appendChild(
```
Comment (DeKleineKobini) [inline extension/scripts/features/itemredirect/ttBazaarRedirect.js:39]: Should it still show the message anymore, as we are searching automatically now?

Comment (Sashank999) [reply extension/scripts/features/itemredirect/ttBazaarRedirect.js:39]: Removed message.

File: extension/scripts/features/bazaarredirect/ttBazaarRedirect.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+		{
+			storage: ["settings.pages.bazaar.redirects"],
+		},
+		null
+	);
+
+	async function addHighlight() {
+		await requireElement("[class*='rowItems_'] [class*='item_']");
+		const params = getSearchParameters();
+		if (params.has("tt_itemid") && !(await checkMobile())) {
```
[the remark this answers is not in the document]
Comment (Sashank999) [reply extension/scripts/features/bazaarredirect/ttBazaarRedirect.js]: Done.

[the remark this answers is not in the document]
Comment (DeKleineKobini) [reply extension/scripts/features/bazaarredirect/ttBazaarRedirect.js]: Mobile checks should be done in `requirements` instead.

**Pull request merged**